backend/src/zac/contrib/dowc/tests/test_dowc_api.py

```diff
@@ -122,7 +122,7 @@ def test_client(self, m):
         token = header.split(" ")[1]
         claims = jwt.decode(token, verify=False)
         self.assertEqual(claims["user_id"], self.user.username)
-
+        self.assertEqual(claims["zds"], {"email": self.user.email})
```

Comment:
```suggestion
        self.assertEqual(claims["email"], self.user.email)
```
and of course, update to gemma-zds-client 1.0 :-)

Reply:
:partying_face: